source/photo.html

```diff
@@ -0,0 +1,183 @@
+            <img src="https://via.placeholder.com/150x150" alt="лайк">
+          </button>
+          <span class="userphotos-list__comment-like">Нравится: 666</span>
+        </li>
+        <li class="userphotos-list__item">
+          <img src="https://via.placeholder.com/150x150" alt="фото">
+          <p class="userphotos-list__comment">
+            <span class="userphotos-list__comment-name">Борис</span>
+            <span class="userphotos-list__comment-time">(20 часов назад)</span>
+            Здесь могла быть ваша цитата о
+            высоком и вечном.
+          </p>
+          <button class="like-button">
+            <img src="https://via.placeholder.com/150x150" alt="лайк">
+          </button>
+          <span class="userphotos-list__comment-like">Нравится: 215</span>
+        </li>
+        <li class="userphotos-list__item">
+          <img src="https://via.placeholder.com/150x150" alt="фото">
+          <p class="userphotos-list__comment">
+            <span class="userphotos-list__comment-name">Сергей</span>
+            <span class="userphotos-list__comment-time">(25 часов назад)</span>
+            Самая кубанская в мире!
+          </p>
+          <button class="like-button">
+            <img src="https://via.placeholder.com/150x150" alt="лайк">
+          </button>
+          <span class="userphotos-list__comment-like">Нравится: 4</span>
+        </li>
+        <li class="userphotos-list__item">
+          <img src="https://via.placeholder.com/150x150" alt="фото">
+          <p class="userphotos-list__comment">
+            <span class="userphotos-list__comment-name">Максим</span>
+            <span class="userphotos-list__comment-time">(40 часов назад)</span>
+            Где снег-то?
+          </p>
+          <button class="like-button">
+            <img src="https://via.placeholder.com/150x150" alt="лайк">
+          </button>
+          <span class="userphotos-list__comment-like">Нравится: 150</span>
+        </li>
+        <li class="userphotos-list__item">
+          <img src="https://via.placeholder.com/150x150" alt="фото">
+          <p class="userphotos-list__comment">
+            <span class="userphotos-list__comment-name">Надежда</span>
+            <span class="userphotos-list__comment-time">(45 часов назад)</span>
+            Всем GM и взаимные лайки!
+          </p>
+          <button class="like-button">
+            <img src="https://via.placeholder.com/150x150" alt="лайк">
+          </button>
+          <span class="userphotos-list__comment-like">Нравится: 2560</span>
+        </li>
+      </ul>
+
+      <p class="photo-text">
+        Затмите их всех!
+        Выкладывайте все, что накопилось
+        в телефоне!
+      </p>
+
+      <div class="sharing">
+        <img src="https://via.placeholder.com/150x150" alt="фото">
+        <div class="sharing-tools">
+          <ul class="sharing-tools__list">
+            <li class="sharing-tools__item sharing-tools--item-cut"></li>
```

Comment:
```suggestion
            <li class="sharing-tools__item sharing-tools__item--cut"></li>
```
и остальные, по аналогии.